### PR TITLE
Make use of RTCPeerConnection.prototype.{getConfiguration, setConfiguration} defensive

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17830)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/269233631)

**Comment by rwaldron** [comment]
Before:



After:

**Comment by soareschen** [comment]
Hmm why does it throw `InvalidAccessError` on your side. I thought it would throw something like "pc.setConfiguration is not a function". It is also weird that testharness doesn't catch the error and just abort remaining tests.

File: webrtc/RTCConfiguration-iceCandidatePoolSize.html [hunk 1]
```diff
@@ -63,17 +66,22 @@
 /*
 Reconfiguration
 */
-const pc = new RTCPeerConnection({});
 
 test(() => {
+  const pc = new RTCPeerConnection({});
```
Comment (soareschen) [inline webrtc/RTCConfiguration-iceCandidatePoolSize.html]: `pc` can be constructed with no argument, i.e. `new RTCPeerConnection()`

File: webrtc/RTCConfiguration-iceCandidatePoolSize.html [hunk 2]
```diff
@@ -88,8 +96,9 @@
 */
 
 test(() => {
+  const pc = new RTCPeerConnection({});
+  assert_idl_attribute(pc, "setConfiguration");
   assert_equals(typeof pc.setConfiguration, "function", "RTCPeerConnection.prototype.setConfiguration is not implemented");
```
Comment (soareschen) [inline webrtc/RTCConfiguration-iceCandidatePoolSize.html]: This become redundant with the `assert_idl_attribute` above. We probably don't want to assert everything too defensively.

**Review by soareschen — CHANGES_REQUESTED** [review]

**Review by soareschen — APPROVED** [review]

**Pull request merged**